gammapy/modeling/models/spectral.py

```diff
@@ -1853,7 +1854,17 @@ def from_dict(cls, data):
             p["value"] for p in data["parameters"] if p["name"] == "alpha_norm"
         ][0]
         if "filename" in data:
-            return cls.read(data["filename"], redshift=redshift, alpha_norm=alpha_norm)
+            if os.path.exists(data["filename"]):
+                return cls.read(
+                    data["filename"], redshift=redshift, alpha_norm=alpha_norm
+                )
+            else:
+                for reference in ["franceschini", "dominguez", "finke"]:
+                    if reference in data["filename"]:
+                        return cls.read_builtin(
+                            reference, redshift=redshift, alpha_norm=alpha_norm
+                        )
```

Comment:
probable should look for the exact file name in case we add mutilple built-in models from the same author.